meilisearch-core/src/raw_indexer.rs

```diff
@@ -70,20 +77,41 @@ impl<A: AsRef<[u8]>> RawIndexer<A> {
     where
         I: IntoIterator<Item = &'s str>,
     {
-        let iter = iter.into_iter();
-        for token in SeqTokenizer::new(iter) {
-            let must_continue = index_token(
-                token,
-                id,
-                indexed_pos,
-                self.word_limit,
-                &self.stop_words,
-                &mut self.words_doc_indexes,
-                &mut self.docs_words,
-            );
-
-            if !must_continue {
-                break;
+        let mut byte_offset = 0;
+        let mut word_offset = 0;
+
+        for s in iter.into_iter() {
```

Comment:
could we rename this? it's not obvious that `iter` is the list of attributes of the document and `s` is the content of 1 attribute.